app/models/poll.rb

```diff
@@ -1,4 +1,9 @@
 class Poll < ApplicationRecord
-  belongs_to :constituency
+  belongs_to :constituency,
+             class_name: "OnsConstituency",
+             foreign_key: "constituency_ons_id",
+             primary_key: "ons_id",
+             inverse_of: "polls",
```

Comment:
Hmm I guess not.  IIUC this is saying that `constituency.polls` will use this `constituency_ons_id` foreign key to find all polls for this constituency
and since constituencies doesn't have any poll id fields, it doesn't need any change here.  Right?

BTW I'm not even sure inverse_of does anything these days.